modules/library/main/src/test/java/org/geotools/styling/visitor/UomRescaleStyleVisitorTest.java

```diff
@@ -293,6 +294,38 @@ protected void visitTextSymbolizerTest(double scaleMetersToPixel, Unit<Length> u
         }
     }
     
+    protected void visitLineSymbolizerTestDynamicDashArray(double scaleMetersToPixel, Unit<Length> uom)
+    {
+        try
+        {
+            UomRescaleStyleVisitor visitor = null;
+            double size = 1;
+			double expectedRescaledSize = Math.floor(computeExpectedRescaleSize(size, scaleMetersToPixel, uom) * 10000.0) / 10000.0;
+			FilterFactory2 filterFactory  = new FilterFactoryImpl();
+            Expression func = filterFactory.function("listMultiply", filterFactory.literal(expectedRescaledSize), filterFactory.literal("5 10"));
+			String expectedDashArray = (String)func.evaluate(null);
+			
+			            StyleBuilder styleBuilder = new StyleBuilder();
+
+            LineSymbolizerImpl lineSymb = (LineSymbolizerImpl) styleBuilder.createLineSymbolizer();
+            lineSymb.setUnitOfMeasure(uom);
+			lineSymb.getStroke().setDashArray(Arrays.asList(filterFactory.literal("5.0 10.0")));
+
+            visitor = new UomRescaleStyleVisitor(scaleMetersToPixel);
+
+            lineSymb.accept(visitor);
+            LineSymbolizer rescaledLineSymb = (LineSymbolizer) visitor.getCopy();
+            String rescaledDynamicDashArray = (String)((Expression)rescaledLineSymb.getStroke().dashArray().get(0)).evaluate(null);
+            
+            assertEquals(expectedDashArray, rescaledDynamicDashArray);
+            assertNotSame(rescaledLineSymb, lineSymb);
```

Comment:
Yes, going round and round it's very frustrating. So is setting up [documented contribution rules](https://github.com/geotools/geotools/blob/master/CONTRIBUTING.md) and seeing them ignored. 

"Please make sure you're following the coding conventions, and otherwise avoid any reformats to the existing code, as they make it harder to review your changes.
[Coding conventions](http://docs.geotools.org/latest/developer/conventions/code/style.html) say:
"developers should use spaces for indentations, not tabulations"